libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/RestClientTest.java

```diff
@@ -576,24 +574,6 @@ public void testQuery() throws Exception {
     }
 
     /**
-     * Testing a search call to the server.
-     * Create new account then look for it using sosl.
-     * @throws Exception
-     */
-    public void testSearch() throws Exception {
-        IdName idNameFirstAccount = createAccount();
-        IdName idNameSecondAccount = createAccount();
-        RestResponse response = restClient.sendSync(RestRequest.getRequestForSearch(TestCredentials.API_VERSION, "find {" + ENTITY_NAME_PREFIX + "}"));
```

Comment:
Why are we killing that test?
If it flaps - we could wait a bit longer (the problem with search is that the update of the search indexes is asynchronous).

Reply:
Yeah - adding time isn't helping much - it's still unpredictable. Based on my discussions with the search team, there's no guaranteed time taken for `SOLR indexing`.

Reply:
Could we still run a search and make sure we don't get a server error - without actually checking the individual records returned? That way we would catch a regression in the soql rest api wrapper if they are ever changed.

Reply:
Yeah, that's a good idea. As long as we don't check the number of records returned, we should be okay.